src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+}
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}: Props) => {
+  const selectedTab = tabs.find((tab) => tab.id === selectedTabId) || tabs[0];
+
+  const handleTabClick = (tab: Tab) => {
```

Comment:
You aren't handling click here.
This function select tab and you use it inside event callback

```suggestion
  const selectTab = (tab: Tab) => {
```